python-package/xgboost/dask.py

```diff
@@ -1287,21 +1288,13 @@ def predict(
         validate_features: bool = True,
         base_margin: Optional[_DaskCollection] = None
     ) -> Any:
-        '''Predict with `data`.
```

Comment:
Why are we removing the docstring for `predict()`?

Reply:
The doc string comes from inheritance as now we support all parameters.